ambari-server/src/test/java/org/apache/ambari/server/topology/BlueprintImplTest.java

```diff
@@ -178,6 +178,8 @@ public void testValidateConfigurations__hostGroupConfigForNameNodeHAPositive() t
     Configuration group2Configuration = new Configuration(group2Props, EMPTY_ATTRIBUTES, configuration);
     expect(group2.getConfiguration()).andReturn(group2Configuration).atLeastOnce();
 
+    org.apache.ambari.server.configuration.Configuration serverConfig = setupConfigurationWithGPLLicense(true);
```

Comment:
It may make sense to convert `serverConfig` to a member variable initialized in `setup()`.